apps/linkedin-to-notion/src/background/messages/notion/providers/notion.provider.ts

```diff
@@ -3,10 +3,17 @@ import type {
   CreatePageResponse,
   DatabaseObjectResponse,
   SearchResponse,
+  UpdatePageResponse,
 } from '@notionhq/client/build/src/api-endpoints';
 
+import { getLinkedinSlug } from '~src/background/shared.utils';
+
+import {
+  databaseSearchResultsToNotionProfileInformation,
+  notionProfileInformationToNotionPage,
+  notionProfileInformationToPageProperties,
+} from '../notion.transformers';
```

Comment:
1. Name of the PR: my bad, I just forgot aha
2. `notion.transformers`: I like that name because I think it better conveys the idea of what goes on (i.e data transformation) which was not as clear before imo. Now that `notionProfileInformationToNotionPage` is under `notion.transformers`, we have more info than we would had we just a function and a file going by the same name.

That being said **you're the expert**, whenever something makes you feel to uncomfortable tech-wise: you should have the last say!

Reply:
I'll say don't worry 😉 
No strong opinion here and I like your rationale